src/utilities/focus.ts

```diff
@@ -1,5 +1,39 @@
+import {
+  findFirstFocusableNode,
+  FOCUSABLE_SELECTOR,
+} from '@shopify/javascript-utilities/focus';
+
 export function handleMouseUpByBlurring({
   currentTarget,
 }: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) {
   currentTarget.blur();
 }
+
+export function nextFocusableNode(
+  node: HTMLElement,
+): HTMLElement | Element | null {
+  let nextElement = node.nextElementSibling;
```

Comment:
This logic will only work if the nextFocusableElement same tree level or lower correct? The following would not pass

```
 <React.Fragment>
    <div><Popover active activator={<div />} onClose={noop} /></div>
     <button id={id} />
</React.Fragment>
```

I think it's fine since the activator will get focus in the end. I can't think of an inexpensive way to do this either. What do you think?